scenarios/chained-cpu-congestion/plan.yaml

```diff
@@ -0,0 +1,34 @@
+---
+phases:
+  - name: Phase1
+    repeat: 10
+    client:
+      workers:
+        - instances: 1
+          duration: 20m
+          delay: 5ms
+    workload:
+      actions:
+        - name: Sleep
+          config:
+            duration: 50ms
+        # Send request to order service
+        - name: HTTPRequest
+          config:
+            url: http://order-service:8080
+            body:
+              actions:    
+                - name: Burn
+                  config:
+                    duration: 10ms
+                # Send request to payment service
+                - name: HTTPRequest
+                  config:
+                    url: http://payment-service:8080
```

Comment:
how would a high cpu utilization in one microservice cause high cpu utilization in a called service?
would it not be likely, if it is busy processing, it either services its clients with higher latency or makes less calls downstream?

also, the readme describes memory leaks, which are not in this plan...

but most importantly, what and where is the cause root (or root causes) that we want to identify/reproduce based on the observed sypmtoms?

Reply:
fixed